torchsupport/modules/dynet.py

```diff
@@ -0,0 +1,37 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as func
+
+class DyNet(nn.Module):
```

Comment:
Cool, I like it. A nice addition would be to have a DyNet Module for 1d and 3d convolution as well.